web-components/src/components/modal/TxSuccessfulModal.tsx

```diff
@@ -59,25 +58,11 @@ const CardItem: React.FC<CardItemProps> = ({
 }) => {
   return (
     <Box sx={{ pt: 5 }}>
-      <Title
-        sx={{
-          textTransform: 'uppercase',
-          fontWeight: 800,
-          letterSpacing: '1px',
-          pb: {
-            xs: 3,
-            sm: 2.25,
-          },
-          fontSize: {
-            sm: 14,
-            xs: 12,
-          },
-        }}
-      >
+      <Label size="sm" sx={{ pb: [3, 2.25] }}>
         {label}
-      </Title>
+      </Label>
 
-      <Description sx={{ fontWeight: 700 }} variant="subtitle2">
+      <Subtitle size="lg" mobileSize="sm">
```

Comment:
the color seems off, it should be #545555:
<img width="546" alt="image" src="https://user-images.githubusercontent.com/3149358/166207637-434a3833-39fc-4d89-956a-978f01fe5309.png">